openmetrics/src/test/java/io/airlift/openmetrics/TestMetricsResource.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.openmetrics;
+
+import io.airlift.http.server.RequestStats;
+import io.airlift.units.Duration;
+import org.testng.annotations.Test;
+import org.weakref.jmx.MBeanExporter;
+
+import javax.management.MBeanServer;
+
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertEquals;
+
+public class TestMetricsResource
+{
+    @Test
+    public void testMetrics()
```

Comment:
We should do some testing of very large numbers of metrics and labels and make sure it's not too slow.